openlibrary/templates/site/body.html

```diff
@@ -22,7 +22,14 @@
       $#print errors (hidden by default as styles are loaded via JS)
       <div class="flash-messages" style="display: none;">
       $for flash in get_flash_messages():
-        <div class="$flash.type"><span>$flash.message</span></div>
+        <div class="$flash.type">
+          <span
+          $if flash.message == "Thank you very much for improving that record!":
```

Comment:
I don't think we want to hard code messages here

Reply:
PR Updated:
Now we are using `get_message("flash_book_updated")`